src/main/java/com/github/firmwehr/gentle/GentleCompiler.java

```diff
@@ -85,7 +51,6 @@ public static void main(String[] args) {
 			runCommand(arguments.path());
 		}
 
-		StaticShutdownCallbackRegistry.invoke();
```

Comment:
Is flushing the output buffer still a concern with this implementation?

Reply:
The `UserOutput` class flushes after writing data.